### PR TITLE
remove-profile-images

### PR DESCRIPTION
**Pull request by charliew-rsi**
Removed references to profile images for current user and authors of articles, removed url image input field in Settings

### CODE REVIEW

File: src/components/Article/Comment.js [hunk 1]
```diff
@@ -15,7 +15,6 @@ const Comment = props => {
         <Link
```
Comment (wesdollar) [inline src/components/Article/Comment.js]: Should this link also be removed? It appears empty at this point.

File: src/components/Header.js [hunk 2]
```diff
@@ -57,7 +57,6 @@ const LoggedInView = props => {
           <Link
```
Comment (wesdollar) [inline src/components/Header.js]: Should this link also be removed? It appears empty at this point.

File: src/components/Article/Comment.js (hunk 1 above)
Comment (charliew-rsi) [reply src/components/Article/Comment.js]: Yes, there is no content inside this Link and the Link itself should not be deleted.

File: src/components/Header.js (hunk 2 above)
Comment (charliew-rsi) [reply src/components/Header.js]: No, this one still has content that (the username listed in text and we probably want to keep that one.

**Pull request merged**